packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -80,39 +81,36 @@ function decryptSpace(space) {
  */
 export function getSpaces(sparkInstance, options = {}) {
   const listOptions = Object.assign({
-    ackFilter: 'noack',
     uuidEntryFormat: true,
     personRefresh: true,
-    latestActivity: '',
     isActive: true,
     lastViewableActivityOnly: true,
     participantAckFilter: 'all',
     deferDecrypt: false,
-    // Only need 2 for One on One conversations
-    participantsLimit: 2
+    allFavorite: true,
+    participantsLimit: -1
```

Comment:
The SDK will set the limit to 0 if we don't pass a variable here. Why? Who knows.